CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ You should also include the user name that made the change.
 - Federation: Add rel attribute to host-meta @mei23
 
 ## 12.110.1 (2022/04/23)
+### NOTE<br>
+Node.js v16.14.0 is recommended for building Misskey v12.110.1<br>
+This version doesn't require Node.js v16.15.0 or later.<br>
+See: https://github.com/misskey-dev/misskey/blob/master/.node-version
```

Comment:
I think this note is misplaced. We also don't have such notes for version 12.110.0 and so on.

Since it caused confusion, I think the only note that would make sense is for the topmost entry, to emphasize that the changes listed there are currently unreleased. In my opinion even that note is unnecessary as the heading already says "unreleased" and the incomplete version number "12.x.x" should make it clear that these changes are unreleased.

Reply:
First of all, if there were good release notes( https://github.com/misskey-dev/misskey/releases ) for each misskey releases in this repo,  In my opinion, we didn't cofuse which Node.js version should be used or not. 

Btw there is a page of "Release Notes"( https://misskey-hub.net/en/docs/releases.html ) in misskey-hub. But it's just a copy of CHANGELOG. In addition, 12.x.x (unreleased) and its Note `From this version, Node 18.0.0 or later is required.` aren't listed in that page. Furthermore, its page is separated from this repo and not linked to.

In conclusion, I will write above highlighted Note in misskey-hub "Release Notes" and want to say this issue caused by a lack release notes in misskey-dev/misskey .